src/hotspot/share/gc/g1/g1RegionToSpaceMapper.cpp

```diff
@@ -70,7 +78,21 @@ class G1RegionsLargerThanCommitSizeMapper : public G1RegionToSpaceMapper {
     guarantee(alloc_granularity >= page_size, "allocation granularity smaller than commit granularity");
   }
 
+  bool committed_range(uint start_idx, size_t num_regions) {
```

Comment:
I would prefer if these two methods were named more like a condition, i.e. `is_range_committed` and `is_range_uncommitted` (optionally exchanging `range` and `[un]committed`) as this makes the verification they are used in read better.